tools/create-changelog-entry.py

```
@@ -0,0 +1,72 @@
+        print("⚠️  No pull request found for the current branch.")
+    return None
+
+
+def main():
+    pr_number = get_pr_number_from_gh()
+    if not pr_number:
+        pr_number = input("Enter the pull request number: ").strip()
+
+    if not pr_number.isdigit():
+        print("❌ Invalid PR number. Please use digits only.")
+        exit(1)
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)